src/schema/posts.ts

```diff
@@ -1764,10 +1767,11 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
       );
 
       const pendingPost = await validateSourcePostModeration(ctx, props);
-      const moderatedPost = await createSourcePostModeration(
-        ctx.con,
-        pendingPost,
-      );
+      const moderatedPost = await createSourcePostModeration({
+        ctx,
+        con: ctx.con,
+        args: pendingPost,
+      });
```

Comment:
Since we're passing `ctx`, we don't need to pass `ctx.con` as well, since we can access it from `ctx` in the function?

Reply:
Right. I thought this was also used somewhere else that utilizes a `transaction`. But yeah, basically about that.